sos/report/__init__.py

```diff
@@ -469,14 +469,15 @@ def _check_container_runtime(self):
             elif crun not in self.policy.runtimes.keys():
                 valid = ', '.join(p for p in self.policy.runtimes.keys()
                                   if p != 'default')
-                raise Exception("Cannot use container runtime '%s': no such "
-                                "runtime detected. Available runtimes: %s"
-                                % (crun, valid))
+                raise Exception(
+                    f"Cannot use container runtime{crun}%s': no such "
```

Comment:
Left over `%s`?

Reply:
Multiline strings with string format, and a few other "complex" methods of string formatting get overlooked by the tool used to implement the f-strings. I forgot about this until after restoring nested lists and doing a search for any remaining `%s` in the repo. 

I've got another commit underway that I started on last night. There are around 200, out of almost 300, lines remaining to review before I push.